### PR TITLE
#N/A - Fix spelling mistake and reorder text

### PR DESCRIPTION
**Pull request by jgalilee**

### CODE REVIEW

File: social/index.html [hunk 1]
```diff
@@ -77,13 +77,13 @@
                 below!
               </p>
               <p class="lead">
+                Once deployed dont forget to read part 2 of the <a href="setup.html">setup guide.</a>
```
Comment (gingermusketeer) [inline social/index.html]: `don't`? 
Should `2` be `two`

**Comment by jgalilee** [comment]
:gift_heart: :whale2:

**Comment by jgalilee** [comment]
:gift_heart: :whale2:

**Comment by gingermusketeer** [comment]
LGTM

File: social/index.html [hunk 2]
```diff
@@ -58,7 +58,7 @@
             <div class="col-sm-12">
               <p class="lead">
                 Social Profile is a Heroku Demo application to showcase the ease
-                at which an application intergrating Facebook can be connected
+                at which an application integrating Facebook can be connected
```
Comment (jmasson) [inline social/index.html]: Should be "with which" rather than "at which".

**Comment by jmasson** [comment]
As discussed, will merge, and I'll do a new PR with a couple tweaks (including above one) later today.

**Pull request merged**